cts-java/src/test/java/org/ga4gh/cts/api/references/ReferenceBasesSearchIT.java

```diff
@@ -0,0 +1,91 @@
+package org.ga4gh.cts.api.references;
+
+import junitparams.JUnitParamsRunner;
+import org.apache.avro.AvroRemoteException;
+import org.ga4gh.ctk.transport.URLMAPPING;
+import org.ga4gh.ctk.transport.protocols.Client;
+import org.ga4gh.cts.api.Utils;
+import org.ga4gh.cts.api.TestData;
+import org.ga4gh.cts.api.reads.ReadsTests;
+import org.ga4gh.methods.*;
+import org.ga4gh.models.Reference;
+import org.ga4gh.models.ReferenceSet;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.ga4gh.cts.api.Utils.aSingle;
+
+/**
+ * Compliance tests relating to searching reference bases.
+ *
+ * @author Maciek Smuga-Otto
+ */
+@Category(ReferencesTests.class)
+@RunWith(JUnitParamsRunner.class)
+public class ReferenceBasesSearchIT {
+
+    private static Client client = new Client(URLMAPPING.getInstance());
+
+    /**
+     * Retrieve all {@link Reference} objects for the default {@link ReferenceSet}. Check
+     * that each returned {@link Reference} has a valid-looking MD5.  (In the future, perhaps this
+     * test should compute the MD5 and compare it to what's returned.)
+     *
+     * @throws AvroRemoteException if there's a communication problem or server exception ({@link GAException})
+     */
+    @Test
+    public void checkMD5ChecksumAppearanceOfAllReferences() throws AvroRemoteException {
```

Comment:
This is still unaddressed, but I won't block on it. I guess you could say the MD5 field of the references is technically related to the reference bases, though this seems like a bit of a stretch.